tests/templates/test_subroutines/test_trotter.py

```diff
@@ -1726,8 +1726,6 @@ def _generate_simple_decomp_trotterize(self, time, order, reverse, args, wires):
                 qml.ControlledPhaseShift(time * arg2, wires=[wires[1], wires[0]]),
                 qml.CNOT([wires[0], wires[1]]),
                 qml.QFT(wires=wires[1:-1]),
```

Comment:
Effectively only one change throughout this test file: Wherever this ansatz is used, remove two of three `QFT`s from it. This does not change the differentiated part but only the static part of the circuit, which makes parameter-shift executions quite expensive.

Reply:
But I think then the QFT would happen on entirely separate wires, right? Maybe that makes the QFT a bit too uncorrelated to what we test? :thinking: